generate-api.sh

```
@@ -0,0 +1,167 @@
+
+# OpenAPI Generator Script for Kinde .NET SDK
+# This script regenerates the .NET client code from the remote OpenAPI specification
+
+set -e
+
+# Configuration
+OPENAPI_SPEC_URL="https://api-spec.kinde.com/kinde-combined-api-specs.yaml"
+TEMP_OUTPUT_DIR="temp-generated"
+OPENAPI_GENERATOR_VERSION="7.0.1"
+OPENAPI_GENERATOR_JAR="openapi-generator-cli.jar"
+API_COPY_DIR="generated-api-files"
```

Shellcheck (0.10.0)

[warning] API_COPY_DIR appears unused. Verify use (or export if used externally).

(SC2034)